cmd/minikube/cmd/start_flags.go

```diff
@@ -349,6 +349,7 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 			},
 			MultiNodeRequested: viper.GetInt(nodes) > 1,
 		}
+		hostOnlyCIDR = strings.Trim(hostOnlyCIDR, "'")
```

Comment:
I think instead of putting this here, we could rewrite this line:


https://github.com/kubernetes/minikube/blob/63faabb0cef1be9b9a2054f0e036bc50a15bda14/cmd/minikube/cmd/start_flags.go#L314

to 

```go
HostOnlyCIDR:      strings.Trim(viper.GetString(hostOnlyCIDR), "'"))
```